var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -89,8 +95,6 @@ def cmake_args(self):
             '-DLAPACK_LIBRARIES=%s' % (spec['lapack'].libs.joined(';')),
             '-DWITH_EXAMPLES=ON',
             self.define_from_variant('BUILD_SHARED_LIBS', 'shared'),
-            self.define_from_variant('USE_HIP', 'rocm'),
-            self.define_from_variant('USE_CUDA', 'cuda'),
```

Comment:
needed for older versions of DBCSR

Reply:
sorry for the blurb, forgot that we (luckily) don't have a version yet for the Spack DBCSR package